src/cypress-api/commands.ts

```diff
@@ -0,0 +1,17 @@
+import { snapshot } from '@chromaui/rrweb-snapshot';
+import type { elementNode } from '@chromaui/rrweb-snapshot';
+// @ts-expect-error will fix when Cypress has its own package
```

Comment:
I'm hoping to punt on making these Cypress files fully type-correct for now (until we create dedicated packages for Cypress and Playwright). To get them correct we'd need to install the Cypress types as a dependency, which means all Playwright users would have Cypress types installed.

Reply:
We could add cypress types as a dev dependency in the meantime to at least avoid these problems in our own code?